build.gradle

```diff
@@ -103,7 +103,7 @@ ext {
     junitVintageVersion = '5.5.2'
     powermockVersion = '2.0.0-beta.5'
     reformLogging = '5.1.0'
-    swagger2Version = '3.0.0-SNAPSHOT'
+    swagger2Version = '2.10.0'
     limits = [
```

Comment:
This should be removed from this PR  swagger2Version = '2.10.0' i should be on dev